lib/spack/spack/util/environment.py

```diff
@@ -25,23 +25,10 @@
 import os
 
 system_paths = ['/', '/usr', '/usr/local']
-suffixes = ['lib', 'lib64', 'include']
-system_dirs = [os.path.join(p, s) for s in suffixes for p in system_paths] + \
-    system_paths
-system_bins = [os.path.join(p, 'bin') for p in system_paths]
 
 
 def filter_system_paths(paths):
-    return [p for p in paths if p not in system_dirs]
```

Comment:
i think you are braking https://github.com/LLNL/spack/pull/2672 Let's keep this function as-is in this PR. If you really want to modify it, please open a separate PR and we discuss it there.